### PR TITLE
test(client): client helper

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

rebase master and replace `logto_access_token_map` key with constant `StorageKeyAccessTokenMap`
test(client): client helper

## Testing

UT

### CODE REVIEW

File: client/helper_test.go [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+func (storage *TestStorage) GetItem(key string) string {
+	return storage.data[key]
+}
+
+func (storage *TestStorage) SetItem(key, value string) {
+	storage.data[key] = value
+}
+
+func TestFetchOidcConfigShouldReturnExpectedOidcConfig(t *testing.T) {
+	wantedOidcConfig := core.OidcConfigResponse{
```
Comment (charIeszhao) [inline client/helper_test.go:48]: nit: `expectedOidcConfig`

File: client/helper_test.go [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+		Token:     "token",
+		Scope:     "scope",
+		ExpiresAt: expiresAt,
+	}
+
+	logtoClient := NewLogtoClient(logtoConfig, testStorage)
+
+	logtoClient.loadAccessTokenMap()
+
+	got := logtoClient.accessTokenMap["@testResource"]
```
Comment (charIeszhao) [inline client/helper_test.go:96]: What about with scopes?

Comment (xiaoyijun) [reply client/helper_test.go:96]: We don't support scopes in V1, so the key will not contain the scope part.

**Review by charIeszhao — APPROVED** [review]
LGTM

**Pull request merged**